Assets/AppCenter/Plugins/AppCenterSDK/Crashes/Shared/Crashes.cs

```diff
@@ -22,9 +22,10 @@ public class Crashes
         // Used by App Center Unity Editor Extensions: https://github.com/Microsoft/AppCenter-SDK-Unity-Extension
         public const string CrashesSDKVersion = "2.5.0";
         private static bool _reportUnhandledExceptions = false;
+        private static bool _enableErrorAttachmentsCallbacks = false;
         private static readonly object _objectLock = new object();
 
-#if UNITY_ANDROID
+#if !UNITY_WSA_10_0
```

Comment:
Can you explain all the pragma changes in this file?

Reply:
Basically that code was android only, but now we say it's okay for android and iOS - so it's okay to let this in all but windows (WSA) now.